blockchain/blockchain-wallet-v4-frontend
# feat(datadog): added support with new header token for datadog

## Description (optional)
This is support for datadog logging 

## Testing Steps (optional)
Run wallet make sure that all API calls contains in request header payload X-Request-ID
